spec/factories/factories.rb

```diff
@@ -11,5 +11,8 @@
   factory :user do
     first_name { Faker::Name.first_name }
     last_name { Faker::Name.last_name }
+    email { Faker::Internet.email }
```

Comment:
Let's extract each factory to its custom file. It's a very good practice to have each factory in its personal file to keep it more organized and clean.